### PR TITLE
SW-4988: FE - Cohort Detail (view/edit) updates

### PR DESCRIPTION
**Pull request by joshbuchea**
This PR includes updates to the Cohort View/New/Edit views.

#### View Cohort

staging terraware io_accelerator_cohorts_1

#### Edit Cohort

staging terraware io_accelerator_cohorts_1 (1)

#### View Cohort

localhost_accelerator_cohorts_1

#### Edit Cohort

localhost_accelerator_cohorts_1_edit

### CODE REVIEW

File: src/scenes/AcceleratorRouter/Cohorts/CohortForm.tsx [hunk 1]
```diff
@@ -102,29 +104,47 @@ export default function CohortForm<T extends CreateCohortRequestPayload | Update
...
-          <Grid item xs={12} sx={{ marginTop: theme.spacing(2) }}>
-            <Dropdown
-              fullWidth={true}
-              label={strings.CURRENT_PHASE}
-              onChange={(value) => updateField('phase', value)}
-              options={currentPhaseDropdownOptions}
-              required
-              selectedValue={localRecord.phase}
+
+          {/* TODO: uncomment this section once createdTime & modifiedTime are available in Cohort records */}
```
Comment (karthikbtf) [inline src/scenes/AcceleratorRouter/Cohorts/CohortForm.tsx:133]: Create a ticket to follow up (here and the other TODOs in this PR)?

**Review by karthikbtf — APPROVED** [review]
Looks good to me. Agree about extracting out those components for reuse.

File: src/scenes/AcceleratorRouter/Cohorts/CohortForm.tsx (hunk 1 above)
Comment (joshbuchea) [reply src/scenes/AcceleratorRouter/Cohorts/CohortForm.tsx:133]: You got it 👍

**Comment by joshbuchea** [comment]
Ok I removed the duplicate components and moved `ProjectField` into `src/components`. Dragging my feet on the rename though because I know I won't be happy with whatever I come up with 😅

**Comment by karthikbtf** [comment]
> Ok I removed the duplicate components and moved `ProjectField` into `src/components`. Dragging my feet on the rename though because I know I won't be happy with whatever I come up with 😅

ProjectField sounds good. ProjectData? It's all good, we can always rename if we want to.

**Review by karthikbtf — APPROVED** [review]
Nice work!

**Pull request merged**